CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Fixed backwards compatibility issue with `SharedApp`. [#6303](https://github.com/mapbox/mapbox-navigation-android/pull/6303)
```

Comment:
We're still going to have to wait another release from the SDK before we can fix this issue https://github.com/mapbox/mapbox-navigation-android/pull/6213/files#r964971123

Reply:
Actually, we may have a way to fix this issue too https://github.com/mapbox/mapbox-navigation-android/pull/6304